indico/modules/events/registration/templates/management/regform_actions/_base.html

```diff
@@ -10,7 +10,18 @@
             </div>
         </div>
         <div class="toolbar right">
-            {% block buttons %}{% endblock %}
+            {% block buttons %}
+                {% if regform.is_scheduled %}
+                    <div class="group">
+                        <a href="#" class="i-button"
```

Comment:
I think a `<button>` would be more appropriate here (assuming it doesn't break any styling etc)